torch_frame/data/tensor_frame.py

```diff
@@ -84,9 +84,6 @@ def __init__(
 
     def validate(self) -> None:
         r"""Validates the :class:`TensorFrame` object."""
-        if len(self.feat_dict) == 0:
```

Comment:
Can we check instead if not only `feat_dict` or `col_names_dict` is empty?

Reply:
It should already be covered by the check `self.feat_dict.keys() != self.col_names_dict.keys()` right below this line.